docs/high_availability_installation_guide.md

```diff
@@ -190,15 +190,13 @@ In ```Harbor.cfg```
 Change the value to your Harbor DB's
```

Comment:
comment for the commit subject:
Move DB configure to central palce => Put DB configurations for normal and HA mode together for readability

Reply:
please change "DB configure" to "DB configuration"
Thanks Yixing very much 👍

Reply:
fixed